<?xml version="1.0"?>
<interface>
  <!-- interface-requires gtk+ 3.8 -->
  <!-- interface-naming-policy toplevel-contextual -->
  <object class="GtkListStore" id="model1">
    <columns>
      <!-- column-name gchararray -->
      <column type="gchararray"/>
    </columns>
    <data>
      <row>
        <col id="0" translatable="yes">Left thumb</col>
      </row>
      <row>
        <col id="0" translatable="yes">Left middle finger</col>
      </row>
      <row>
        <col id="0" translatable="yes">Left ring finger</col>
      </row>
      <row>
        <col id="0" translatable="yes">Left little finger</col>
      </row>
      <row>
        <col id="0" translatable="yes">Right thumb</col>
      </row>
      <row>
        <col id="0" translatable="yes">Right middle finger</col>
      </row>
      <row>
        <col id="0" translatable="yes">Right ring finger</col>
      </row>
      <row>
        <col id="0" translatable="yes">Right little finger</col>
      </row>
    </data>
  </object>
  <object class="GtkAssistant" id="assistant">
    <property name="border_width">12</property>
    <property name="title" translatable="yes">Enable Fingerprint Login</property>
    <property name="icon_name">system-users</property>
    <property name="visible">False</property>
    <child>
      <placeholder/>
    </child>
    <child>
      <object class="GtkVBox" id="page1">
        <property name="visible">True</property>
        <property name="border_width">12</property>
        <property name="orientation">vertical</property>
        <property name="spacing">12</property>
        <child>
          <object class="GtkHBox" id="hbox74">
            <property name="visible">True</property>
            <property name="spacing">6</property>
            <child>
              <object class="GtkImage" id="image">
                <property name="visible">True</property>
                <property name="stock">gtk-dialog-info</property>
                <property name="icon-size">6</property>
              </object>
              <packing>
                <property name="expand">False</property>
                <property name="position">0</property>
              </packing>
            </child>
            <child>
              <object class="GtkLabel" id="intro-label">
                <property name="visible">True</property>
                <property name="label">To enable fingerprint login, you need to save one of your fingerprints, using the Acme Foobar 5000.</property>
                <property name="wrap">True</property>
              </object>
              <packing>
                <property name="expand">False</property>
                <property name="fill">False</property>
                <property name="position">1</property>
              </packing>
            </child>
          </object>
          <packing>
            <property name="position">0</property>
          </packing>
        </child>
        <child>
          <object class="GtkVBox" id="vbox70">
            <property name="visible">True</property>
            <property name="orientation">vertical</property>
            <child>
              <object class="GtkRadioButton" id="radiobutton1">
                <property name="label" translatable="yes">_Right index finger</property>
                <property name="visible">True</property>
                <property name="can_focus">True</property>
                <property name="receives_default">False</property>
                <property name="use_underline">True</property>
                <property name="active">True</property>
                <property name="draw_indicator">True</property>
              </object>
              <packing>
                <property name="expand">False</property>
                <property name="fill">False</property>
                <property name="position">0</property>
              </packing>
            </child>
            <child>
              <object class="GtkRadioButton" id="radiobutton2">
                <property name="label" translatable="yes">_Left index finger</property>
                <property name="visible">True</property>
                <property name="can_focus">True</property>
                <property name="receives_default">False</property>
                <property name="use_underline">True</property>
                <property name="draw_indicator">True</property>
                <property name="group">radiobutton1</property>
              </object>
              <packing>
                <property name="expand">False</property>
                <property name="fill">False</property>
                <property name="position">1</property>
              </packing>
            </child>
            <child>
              <object class="GtkHBox" id="radiobutton3-hbox">
                <property name="visible">True</property>
                <property name="spacing">6</property>
                <child>
                  <object class="GtkRadioButton" id="radiobutton3">
                    <property name="visible">True</property>
                    <property name="label" translatable="yes">_Other finger:</property>
                    <property name="can_focus">True</property>
                    <property name="receives_default">False</property>
                    <property name="use_underline">True</property>
                    <property name="draw_indicator">True</property>
                    <property name="group">radiobutton1</property>
                  </object>
                  <packing>
                    <property name="expand">False</property>
                    <property name="fill">False</property>
                  </packing>
                </child>
                <child>
                  <object class="GtkComboBox" id="finger_combobox">
                    <property name="visible">True</property>
                    <property name="sensitive">False</property>
                    <property name="model">model1</property>
                    <child>
                      <object class="GtkCellRendererText" id="renderer1"/>
                      <attributes>
                        <attribute name="text">0</attribute>
                      </attributes>
                    </child>
                  </object>
                  <packing>
                    <property name="expand">False</property>
                    <property name="fill">False</property>
                  </packing>
                </child>
              </object>
              <packing>
                <property name="expand">False</property>
                <property name="position">2</property>
              </packing>
            </child>
          </object>
          <packing>
            <property name="position">1</property>
          </packing>
        </child>
      </object>
    </child>
    <child>
      <object class="GtkVBox" id="page2">
        <property name="visible">True</property>
        <property name="orientation">vertical</property>
        <child>
          <object class="GtkLabel" id="enroll-label">
            <property name="visible">True</property>
            <property name="label">In order to save your fingerprints, you need to swipe your thumb on the "Acme foobar" device.</property>
            <property name="wrap">True</property>
          </object>
          <packing>
            <property name="expand">False</property>
            <property name="fill">False</property>
            <property name="position">0</property>
          </packing>
        </child>
        <child>
          <object class="GtkFlowBox" id="enroll-flowbox">
            <property name="visible">True</property>
            <property name="margin-top">24</property>
            <property name="margin-bottom">24</property>
            <property name="row-spacing">40</property>
            <property name="column-spacing">100</property>
            <property name="homogeneous">True</property>
            <property name="halign">center</property>
            <property name="valign">center</property>
            <property name="min-children-per-line">1</property>
            <property name="max-children-per-line">5</property>
            <property name="selection-mode">none</property>
            <property name="focus-on-click">False</property>
            <property name="can-focus">False</property>
          </object>
          <packing>
            <property name="expand">False</property>
            <property name="fill">False</property>
            <property name="position">1</property>
          </packing>
        </child>
        <child>
          <object class="GtkLabel" id="status-label">
            <property name="visible">True</property>
            <property name="wrap">True</property>
          </object>
          <packing>
            <property name="expand">False</property>
            <property name="fill">False</property>
            <property name="position">2</property>
          </packing>
        </child>
      </object>
    </child>
    <child>
      <object class="GtkLabel" id="page3">
        <property name="visible">True</property>
        <property name="label" translatable="yes">Your fingerprint was successfully saved. You should now be able to log in using your fingerprint reader.</property>
        <property name="wrap">True</property>
      </object>
      <packing>
        <property name="page_type">summary</property>
      </packing>
    </child>
  </object>
  <object class="GtkSizeGroup" id="sizegroup">
    <property name="mode">both</property>
    <widgets>
      <widget name="radiobutton1"/>
      <widget name="radiobutton2"/>
      <widget name="radiobutton3-hbox"/>
    </widgets>
  </object>
</interface>
